projects/flyout/src/modules/flyout/fixtures/flyout-sample.component.fixture.html

```diff
@@ -0,0 +1,16 @@
+{{ data?.name }}
+<iframe
+  src="https://developer.blackbaud.com/skyux/"
+  *ngIf="data?.showIframe"
```

Comment:
Would you mind putting the structural directives on the first line? `<iframe *ngIf...`

Reply:
I can - that was prettier doing its thing.

Reply:
Fixed

Reply:
Note: prettier moves it to the second line - but I fixed the ordering.